.mk/build.mk

```diff
@@ -19,7 +19,7 @@ include $(foreach module, $(modules), ./$(module)/build.mk)
 BIN_DIR = $(PWD)/build/dist
 VERSION = $(shell git describe --tags --always)
 # Temporary while image is not uploaded to ligato docker hub repo
-VPP_AGENT=ligato/vpp-agent:v3.1.0
+VPP_AGENT=artgl/vpp-agent:v3.2.0
```

Comment:
Could you create an issue in this repository to change this repository back when ligato guys provide a new release with a new vpp?